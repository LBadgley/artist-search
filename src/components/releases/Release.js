import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';

function Release({ releaseArt, releaseName, releaseId }) {
  return (
    <Link to={`/songs/${releaseId}`}>
      <img width="100px" src={`${releaseArt}`} />
      {releaseName}
    </Link>
  );
}

Release.propTypes = {
  releaseName: PropTypes.string,
  releaseArt: PropTypes.string,
  releaseId: PropTypes.string
};

export default Release;
