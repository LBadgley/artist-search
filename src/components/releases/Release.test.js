import React from 'react';
import { shallow } from 'enzyme';
import Release from './Release';

describe('release test', () => {
  it('renders the release', () => {
    const wrapper = shallow(<Release />);
    expect(wrapper).toMatchSnapshot();
  });
});
