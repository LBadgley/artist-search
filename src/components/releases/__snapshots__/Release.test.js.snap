// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`release test renders the release 1`] = `
<Link
  to="/songs/undefined/undefined"
>
  <img
    className="releasesStyles"
    src="undefined"
    width="100px"
  />
</Link>
`;
